cpp/src/tsne/tsne.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cuML.hpp"
+
+#pragma once
+
+namespace ML {
+
+void TSNE_fit(const cumlHandle &handle, const float *X, float *Y, const int n,
```

Comment:
Still need a doxygen comment here since it's the definition for the public API (will be included in any `include` dirs)

Reply:
Ok I'll copy over the cpp docs